### PR TITLE
Made two more configuration guides flex-ready

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

File: configuration/environments.rst [hunk 1]
```diff
@@ -20,206 +20,136 @@ Different Environments, different Configuration Files
...
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd
-                http://symfony.com/schema/dic/webprofiler
-                http://symfony.com/schema/dic/webprofiler/webprofiler-1.0.xsd">
-
-            <imports>
-                <import resource="config.xml" />
-            </imports>
+            // ...
+            $confDir = dirname(__DIR__).'/config';
```
Comment (yceruto) [inline configuration/environments.rst]: `$this->getProjectDir()`? in line with the latest changes https://github.com/symfony/recipes/pull/244.

File: configuration/environments.rst [hunk 2]
```diff
@@ -333,50 +245,77 @@ The best way to accomplish this is via a new environment called, for example,
 
     .. code-block:: php
 
-        // app/config/config_benchmark.php
-        $loader->import('config_prod.php');
-
+        // config/benchmark/web_profiler.php
```
Comment (yceruto) [inline configuration/environments.rst]: `config/packages/benchmark/web_profiler.php`?

File: configuration/external_parameters.rst [hunk 3]
```diff
@@ -18,22 +18,30 @@ the variables you want to use enclosed between ``env()``. Their actual values
...
-``%env(DATABASE_HOST)%`` in your configuration files:
+For example, when installing the ``doctrine`` recipe, database configuration is
+put in a ``DATABASE_URL`` environment variable:
+
+.. code-block:: bash
+
+    # .env
+    DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?charset=utf8mb4&serverVersion=5.7"
+
+This variable is referenced in the serivce container configuration using
```
Comment (yceruto) [inline configuration/external_parameters.rst]: typo `service`

File: configuration/environments.rst [hunk 4]
```diff
@@ -20,206 +20,136 @@ Different Environments, different Configuration Files
...
-        $container->loadFromExtension('web_profiler', array(
-            'toolbar' => true,
+Take the framework package, installed by default, as an example:
 
-            // ...
-        ));
+* Loaded in all environments, ``config/packages/framework.yaml`` configures the
+  framework with some ``secret`` setting;
+* In the **prod** environment, nothing extra will be set as there is no
+  ``config/packages/prod`` directory;
```
Comment (javiereguiluz) [inline configuration/environments.rst]: `config/packages/prod` -> `config/packages/prod/`

**Review by javiereguiluz — APPROVED** [review]
Nice!

**Comment by wouterj** [comment]
Thanks @javiereguiluz @yceruto, fixed all comments you made!

**Pull request merged**